src/lighteval/main_accelerate.py

```diff
@@ -20,63 +20,187 @@
+    from lighteval.utils.utils import boolstring_to_bool
 
+    accelerator = Accelerator(kwargs_handlers=[InitProcessGroupKwargs(timeout=timedelta(seconds=3000))])
 
-@htrack()
-def main(args):
-    env_config = EnvConfig(token=TOKEN, cache_dir=args.cache_dir)
+    env_config = EnvConfig(token=TOKEN, cache_dir=cache_dir)
     evaluation_tracker = EvaluationTracker(
-        output_dir=args.output_dir,
-        save_details=args.save_details,
-        push_to_hub=args.push_to_hub,
-        push_to_tensorboard=args.push_to_tensorboard,
-        public=args.public_run,
-        hub_results_org=args.results_org,
+        output_dir=output_dir,
+        save_details=save_details,
+        push_to_hub=push_to_hub,
+        push_to_tensorboard=push_to_tensorboard,
+        public=public_run,
+        hub_results_org=results_org,
     )
     pipeline_params = PipelineParameters(
         launcher_type=ParallelismManager.ACCELERATE,
         env_config=env_config,
-        job_id=args.job_id,
-        dataset_loading_processes=args.dataset_loading_processes,
-        custom_tasks_directory=args.custom_tasks,
-        override_batch_size=args.override_batch_size,
-        num_fewshot_seeds=args.num_fewshot_seeds,
-        max_samples=args.max_samples,
-        use_chat_template=args.use_chat_template,
-        system_prompt=args.system_prompt,
+        job_id=job_id,
+        dataset_loading_processes=dataset_loading_processes,
+        custom_tasks_directory=custom_tasks,
+        override_batch_size=override_batch_size,
+        num_fewshot_seeds=num_fewshot_seeds,
+        max_samples=max_samples,
+        use_chat_template=use_chat_template,
+        system_prompt=system_prompt,
     )
 
-    model_config = create_model_config(
-        use_chat_template=args.use_chat_template,
-        override_batch_size=args.override_batch_size,
-        model_args=args.model_args,
-        model_config_path=args.model_config_path,
-        accelerator=accelerator,
-    )
+    if model_args.endswith(".yaml"):
+        with open(model_args, "r") as f:
+            config = yaml.safe_load(f)["model"]
+
+        # Creating the multichoice space parameters
+        # We need to take into account possible conversion issues from our different input formats
+        multichoice_continuations_start_space = boolstring_to_bool(
+            config["generation"]["multichoice_continuations_start_space"]
+        )
+
+        if multichoice_continuations_start_space is not None:
+            if multichoice_continuations_start_space:
+                logger.info(
+                    "You set `multichoice_continuations_start_space` to true. This will force multichoice continuations to use a starting space"
+                )
+            else:
+                logger.info(
+                    "You set `multichoice_continuations_start_space` to false. This will remove a leading space from multichoice continuations, if present."
+                )
```

Comment:
Isn't that already managed in the config class init?

Reply:
not sure, this was in the config class create function, but this function isn't used anymore (we don't need to route the model args to the correct config file )

Reply:
let's remove deduped code